platform/ios/src/MGLAnnotationView.h

```diff
@@ -40,6 +40,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, assign, getter=isFlat) BOOL flat;
 
+/**
+ Setting this property to YES will cause the annotation to shrink as it approaches the horizon and grow as it moves away from the
+ horizon when the associated map view is tilted.
+ */
+@property (nonatomic, assign, getter=isScaled) BOOL scaled;
```

Comment:
“Scaling” happens for a lot of reasons on iOS. This property should be more descriptively named so that people don’t expect it to relate to the device’s pixel ratio or `UIImage.scale` somehow. Maybe `scalesByViewingDistance` or something?